src/api_authentication_app_skeleton/src/actions/api/password_resets/new.cr

```diff
@@ -0,0 +1,9 @@
+class Api::PasswordResets::New < ApiAction
+  include Auth::PasswordResets::Base
+
+  param token : String
+
+  get "/api/password_resets/:user_id" do
+    text "Please configure password reset page on client."
+  end
+end
```

Comment:
I think this can be removed and instead we can add something to the authentication guide: https://luckyframework.org/guides/authentication/

Reply:
Okay but if I remove this action it raises an error since the email templates at `/src/base_authentication_app_skeleton/src/emails/templates/password_reset_request_email/text.ecr.ecr` reference the action:

```
Please reset your password:

<%="<"%>%= PasswordResets::New.url(@user.id, @token) %>
```

Reply:
I'm beginning to think that we should remove PasswordResets for now, or start them in a separate PR after this one is removed. It's not quite there IMO and I'd rather not have it block the rest of this PR

It would be a bit of a pain for people to implement themselves, but I think it'd also be a pain to figure out how to integrate with it as it is now. So I think we should hold off on it

What do you think?